pkgs/os-specific/linux/kernel/patches.nix

```diff
@@ -34,6 +34,11 @@
       patch = ./modinst-arg-list-too-long.patch;
     };
 
+  selinux_empty_opts =
+    { name = "selinux-empty-opts";
+      patch = ./selinux-empty-opts.patch;
```

Comment:
Why can't we fetchurl https://lore.kernel.org/selinux/20230911142358.883728-1-omosnace@redhat.com/raw?

Reply:
Done!